po/lt.po

```diff
@@ -0,0 +1,314 @@
+msgid "the file naming specification by freedesktop.org"
+msgstr "rinkmenos užvadinimo specifikacija pagal freedesktop.org"
+
+#: src/Views/FilesView.vala:32
+msgid "Create a new entry"
+msgstr "Kurti naują įvestį"
+
+#: src/Views/FilesView.vala:37
+msgid "Light"
+msgstr "Šviesus"
+
+#: src/Views/FilesView.vala:38
+msgid "Dark"
+msgstr "Tamsus"
+
+#: src/Views/FilesView.vala:39 src/CategoryChooser.vala:108
+msgid "System"
+msgstr "Sistema"
+
+#: src/Views/FilesView.vala:42
+msgid "Style"
+msgstr "Stilius"
+
+#. TRANSLATORS: %s will be replaced by the app name (Pin It!)
+#: src/Views/FilesView.vala:47
+#, c-format
+msgid "About %s…"
+msgstr "Apie %s…"
+
+#: src/Views/FilesView.vala:51
+msgid "Preferences"
+msgstr "Nustatymai"
+
+#: src/Views/FilesView.vala:77
+msgid "No valid app entries found"
+msgstr "Nebuvo rasta tinkamos programėlės įvesties"
+
+#: src/Views/FilesView.vala:78
+msgid "If you've never created one, click the + button on the top."
+msgstr "Jeigu nemėginai įgyvendinti spausk Plius mygtuką viršuje."
+
+#: src/Views/FilesView.vala:130
+msgid "Delete…"
+msgstr "Ištrinti... ."
+
+#: src/Views/FilesView.vala:136
+#, c-format
+msgid "Are you sure you want to delete “%s”?"
+msgstr "Ar esi tikras, kad nori ištrinti “%s”?"
+
+#: src/Views/FilesView.vala:137
+msgid "This removes the app from the launcher."
+msgstr "Tai pašalina programėlę iš paleidiklio."
+
+#: src/Views/FilesView.vala:140
+msgid "Delete"
+msgstr "Ištrinti"
+
+#: src/Views/FilesView.vala:152
+#, c-format
+msgid "Could not delete file “%s”"
+msgstr "Neišeina ištrinti rinkmenos  “%s”"
+
+#: src/CategoryChooser.vala:97
+msgid "Audio & Video"
+msgstr "Audiovizualinių įrašų programa"
+
+#: src/CategoryChooser.vala:98
+msgid "Audio"
+msgstr "Audio įrašų programa"
+
+#: src/CategoryChooser.vala:99
+msgid "Video"
+msgstr "Vaizdo įrašų programa"
+
+#: src/CategoryChooser.vala:100
+msgid "Development"
+msgstr "Kūrėjo ir plėtros programa"
+
+#: src/CategoryChooser.vala:101
+msgid "Education"
+msgstr "Programa mokinimuisi"
+
+#: src/CategoryChooser.vala:102
+msgid "Game"
+msgstr "Žaidybinė programa"
+
+#: src/CategoryChooser.vala:103
+msgid "Graphics"
+msgstr "Grafikos programa"
+
+#: src/CategoryChooser.vala:104
+msgid "Network"
+msgstr "Ryšių ir tinklo programa"
+
+#: src/CategoryChooser.vala:105
+msgid "Office"
+msgstr "Biuro programa"
+
+#: src/CategoryChooser.vala:106
+msgid "Science"
+msgstr "Mokslinių tyrimų programa"
+
+#: src/CategoryChooser.vala:107
+msgid "Settings"
+msgstr "Nustatymų programa"
+
+#: src/CategoryChooser.vala:109
+msgid "Utility"
+msgstr "Paslaugų programa"
+
+#: src/AboutWindow.vala:32
+msgid ""
+"Create the shortcut to your favorite portable apps into your app launcher"
+msgstr ""
+"Sukurti po greitąją nuorodą mėgstamiausioms perkeliamoms programėlėms jūsų "
+"programų paleidiklyje"
+
+#. TRANSLATORS: Replace with your name; don't translate literally
+#: src/AboutWindow.vala:42
+msgid "translator-credits"
+msgstr "išvertimo-įnašas"
```

Comment:
Please read the comment for translators above in L303:

```
TRANSLATORS: Replace with your name; don't translate literally
```

Or alternatively, keep it blank if you don't like to be credited here.

Reply:
Oh there exists comments!
I need to Configure my App, because it does not show them and it goes straight to the Point.
Now the catch. Because You have found some errors, then now both of our Contributions should be listed in there!